### PR TITLE
Add link to alternative account creation methods

### PR DESCRIPTION
**Pull request by montehurd**
https://phabricator.wikimedia.org/T154728

req mov

### CODE REVIEW

File: Wikipedia/Localizations/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -49,6 +49,9 @@
 "account-creation-captcha-required" = "CAPTCHA verification required.";
 "account-creation-captcha-retry" = "Incorrect entry. Please try again.";
 "account-creation-captcha-obtaining" = "Obtaining a new CAPTCHA...";
+"account-creation-captcha-title" = "CAPTCHA security check";
+"account-creation-captcha-cannot-see-image" = "Can't see the image? $1";
+"account-creation-captcha-request-account" = "Request an account.";
 "account-creation-logging-in" = "Logging in...";
```
Comment (montehurd) [inline Wikipedia/Localizations/en.lproj/Localizable.strings]: Quick note for Andy: localized strings added to this file get automatically added to translatewiki.net. We periodically get translations back - about weekly, as PRs, which are then tested (occasionally translators will accidentally add things like html comments which are fine if the string was going to used with html for a web page but which don't make sense for the native app).

Here's a link to the iOS TWN page: https://translatewiki.net/w/i.php?title=Special:Translate&group=out-wikimedia-mobile-wikipedia-ios&filter=translated&language=de&action=translate
Usually we don't have to do anything with this, but it's nice to see where the translations actually happen.

File: Wikipedia/Localizations/qqq.lproj/Localizable.strings [hunk 2]
```diff
@@ -59,6 +59,9 @@
 "account-creation-captcha-required" = "Alert message shown when account creation require a captcha verification";
 "account-creation-captcha-retry" = "Alert message shown when the captcha the user entered did not match the captcha image";
 "account-creation-captcha-obtaining" = "Alert shown when user wants a new captcha when creating account";
+"account-creation-captcha-title" = "Title for account creation CAPTCHA interface";
```
Comment (montehurd) [inline Wikipedia/Localizations/qqq.lproj/Localizable.strings]: Any time a new translation string is added this is the place you must add a description of the string. This is presented to translators on TWN to provide context.

File: Wikipedia/Code/WMFAccountCreationViewController.swift [hunk 3]
```diff
@@ -97,8 +101,34 @@ class WMFAccountCreationViewController: UIViewController, WMFCaptchaViewControll
...
+        
+        // Reminder: used a label instead of a button for subtitle because of multi-line string issues with UIButton.
+        captchaSubtitleLabel.attributedText = getCaptchaSubtitleAttributedString()
+        captchaSubtitleLabel.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(self.requestAnAccountTapped(_:))))
+
+        view.wmf_configureSubviewsForDynamicType()
     }
 
+    func requestAnAccountTapped(_ recognizer: UITapGestureRecognizer) {
+        wmf_openExternalUrl(URL.init(string: "https://en.wikipedia.org/wiki/Wikipedia:Request_an_account"))
```
Comment (amrox) [inline Wikipedia/Code/WMFAccountCreationViewController.swift]: I guess this page only exists on the english site? Or is it worth trying to localize?

Comment (montehurd) [inline Wikipedia/Code/WMFAccountCreationViewController.swift]: Good point. I just double-checked with Carolyn and she said this is ok for now (until we can figure out the other pages).

**Comment by montehurd** [comment]
@joewalsh this should be good to go.

File: Wikipedia/Code/WMFAccountCreationViewController.swift [hunk 4]
```diff
@@ -97,8 +101,34 @@ class WMFAccountCreationViewController: UIViewController, WMFCaptchaViewControll
...
+    }
+        
+    fileprivate func getCaptchaSubtitleAttributedString() -> NSAttributedString {
```
Comment (joewalsh) [inline Wikipedia/Code/WMFAccountCreationViewController.swift]: stylistically, this should probably be 
```
fileprivate var captchaSubtitleAttributedString: NSAttributedString {
    get {
        -implementation here-
    }
}
```

**Comment by joewalsh** [comment]
!jenkins test

File: Wikipedia/Code/WMFAccountCreationViewController.swift (hunk 4 above)
Comment (montehurd) [inline Wikipedia/Code/WMFAccountCreationViewController.swift]: Fixed!

**Pull request merged**